### PR TITLE
Switch to `import_granularity: item`

### PR DESCRIPTION
**Pull request by thomaseizinger**
Putting each import statement on a dedicated line makes it a lot less
likely to create merge conflicts when items are removed / added because
Git resolves conflicts on a line-by-line basis.

### CODE REVIEW

**Review by luckysori — APPROVED** [review]
RemindMe! One Week "Revert this change".

File: daemon/src/auto_rollover.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (thomaseizinger) [inline daemon/src/auto_rollover.rs]: Accidentially committed this!

File: daemon/src/supervisor.rs [hunk 2]
```diff
@@ -0,0 +1,91 @@
+use crate::Tasks;
+use async_trait::async_trait;
+use std::fmt;
+use xtra::Address;
+use xtra::Context;
+use xtra_productivity::xtra_productivity;
+
+/// A supervising actor reacts to messages from the actor it is supervising and restarts it based on
+/// a given policy.
+pub struct Actor<T, R> {
```
Comment (thomaseizinger) [inline daemon/src/supervisor.rs]: And this!

**Comment by klochowicz** [comment]
I'm happy with it as long as we can revert it back after the conflict-prone stuff gets in.  :)

**Review by klochowicz — APPROVED** [review]

**Pull request merged**